tsconfig.build.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "./tsconfig.json",
+  "exclude": ["./generated/**/hardhat.d.ts"],
+  "include": ["./generated/**/*.ts", "./generated/**/node_modules/**/*.ts"],
```

Comment:
I doubt that we need these exclude and include lines

Reply:
These are good to have. No need to compile unwanted files.

Reply:
I doubt hardhat.d.ts would be much, haha

Reply:
No, It won't but if there is something we can avoid we should.

Reply:
Hmm, I test already, if a file is included because of the "files" entry, it will be included in the compiler output despite being included in "exclude" entry

Reply:
so again, these two files are unncessary I think, unless they are for other purposes